pandas-stubs/io/sql.pyi

```diff
@@ -33,6 +34,7 @@ _SQLStatement: TypeAlias = (
     | sqlalchemy.sql.expression.TextClause
     | sqlalchemy.sql.Select
     | FromStatement
+    | UpdateBase
```

Comment:
Instead of importing `UpdateBase`, use the pattern as in lines 33 and 34, and place `sqlalchemy.sql.expression.UpdateBase` here